packages/core/content-releases/server/src/services/release.ts

```diff
@@ -181,9 +183,9 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
     releaseId: GetReleaseActions.Request['params']['releaseId'],
     query?: GetReleaseActions.Request['query']
   ) {
-    const result = await strapi.entityService.findOne(RELEASE_MODEL_UID, releaseId);
+    const release = await strapi.entityService.findOne(RELEASE_MODEL_UID, releaseId);
```

Comment:
It looks like release only used to check that it's not used. So you could use the fields param and give it an empty array to improve the query performance since we don't need the data